mesop/components/html/html.ng.html

```diff
@@ -0,0 +1 @@
+<div [innerHTML]="config().getHtml()" [style]="getStyle()"></div>
```

Comment:
So this is no longer rendered in an iframe? I see some iframe changes in this PR still. How are those related?

Reply:
Yeah, no longer rendered in an iframe. The iframe changes were left over by accident - I've removed them.